docs/web-console-guide_5/master.adoc

```diff
@@ -48,7 +48,24 @@ include::topics_5/installing-web-console-on-openshift.adoc[leveloffset=+3]
 include::topics_5/installing-web-console-on-openshift.adoc[leveloffset=+3]
 :ocp-311!:
 :web-console-guide:
-include::topics_5/web-openshift-troubleshoot.adoc[leveloffset=+3]
+==== Troubleshooting a {WebName} installation on OpenShift
+
+The following steps discuss common techniques for troubleshooting a {WebName} installation in an OpenShift Container Platform environment. These instructions focus on issues specific to the {WebName}.
```

Comment:
Second sentence is not accurate. These instructions do not focus on issues specific to the web console. They focus on issues specific to installing the web console on OpenShift. However, since this is already mentioned in the title, I would say that the second sentence is fluff and should be removed.